### PR TITLE
feat: enable GitHub integration

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

**Comment by raisedadead** [comment]
Definitely do not want the comments.

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -20,7 +20,11 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v1
         with:
+          record: true
           build: npm run build
+        env:
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (ojeytonwilliams) [inline .github/workflows/cypress.yml:27]:
```suggestion
        env:
          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          record: true
          build: npm run build
```
The order meant some `with` entries were being interpreted as `env` entries.

Also, I saw this when setting up the CYPRESS_RECORD_KEY: 
> Secrets are not passed to workflows that are triggered by a pull request from a fork.

presumably that means PR runs won't get recorded.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Comment by ojeytonwilliams** [comment]
I did not get any comments when I did this on my fork: https://github.com/ojeytonwilliams/freeCodeCamp/pull/7

Could there be a dashboard setting that's enabling them?

**Comment by raisedadead** [comment]
This PR is more of a spike to gauge: if we even want to use https://cypress.io for storing recordings, it will be useful to check when something breaks. Currently CI have logs only.

I will add you to the dashboard today.

I have used this in past (on my website), its good.

**Comment by raisedadead** [comment]
Closing in favor of https://github.com/freeCodeCamp/freeCodeCamp/pull/38678 for convention

**Pull request closed without merging**